api/admin.py

```diff
@@ -5,6 +5,7 @@
 from django.utils.translation import gettext_lazy as _
 from django.utils.html import format_html_join
 from django.utils.safestring import mark_safe
+from django.db import models as dj_models
```

Comment:
Where dj_models is used?

Reply:
Good find. I will remove this on the next PR if it's okay.